src/components/PublicPursuances/PublicPursuanceList.js

```diff
@@ -1,13 +1,48 @@
+// dumb component
+
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { Link } from 'react-router-dom';
 import * as postgrest from '../../api/postgrest';
 
 
 class PublicPursuanceList extends Component {
+
+  orderByDateDesc = (p1, p2) => {
+    p1["created_parsed"] = Date.parse(p1.created);
+    p2["created_parsed"] = Date.parse(p2.created);
+    return p2.created_parsed - p1.created_parsed;
+  }
+  orderByDateAsc = (p1, p2) => {
+    return p1.created_parsed - p2.created_parsed;
+  }
+  orderByNameAsc = (p1, p2) => {
+    return p1.name.localeCompare(p2.name);
+  }
+  orderByNameDesc = (p1, p2) => {
+    return p2.name.localeCompare(p1.name);
```

Comment:
@thevaleriemack Please do the comparison after a `.toLowerCase()` on each `name` so that `a pursuance with this name` doesn't show up after `This pursuance name` due to capitalization differences.